unit-tests/func/hw-errors/l500/test-while-streaming.cpp

```diff
@@ -0,0 +1,24 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
If the only diff between this and the previous is streaming, let's combine into the same test?